### ISSUE TITLE
[Bug]: Ultimate SD Upscale crashing after 1.6.0 merge

### ISSUE THREAD

**Issue by Convexity-ai** [lshqqytiger/stable-diffusion-webui-directml#263]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

When using the [Ultimate SD Upscale extension](https://github.com/Coyote-A/ultimate-upscale-for-automatic1111), the extension will cause SD will crash after processing the first part of an image.

It's been difficult to troubleshoot the cause of this as the crash doesn't generate any error, instead SD will completely freeze and the only option is to close the command prompt window.

I've not seen mention of this issue upstream nor do I know if it's up to the extension dev to resolve their side. This extension was working on version 1.5.2 and still works on the latest version of [SD.Next](https://github.com/vladmandic/automatic) using the DirectML backend.

Further information below.

### Steps to reproduce the problem

1. Install Ultimate SD Upscale extension.
2. Go to 'img2img' tab and load an image.
3. Under scripts, select 'Ultimate SD upscale'
4. Click 'Generate'

### What should have happened?

Ultimate SD Upscale should split the image into smaller images defined by the 'Tile width' and 'Tile height' values and rebuild / upscale the image based on generation parameters.

### Sysinfo

[sysinfo-2023-09-05-04-14.txt](https://github.com/lshqqytiger/stable-diffusion-webui-directml/files/12518727/sysinfo-2023-09-05-04-14.txt)

### What browsers do you use to access the UI ?

Microsoft Edge

### Console logs

```Shell
C:\Program Files\stable-diffusion-webui-directml>webui-user.bat
venv "C:\Program Files\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.6.0-296-g39e6d26f
Commit hash: 39e6d26f3b66805677cc3b5c2bc3bd624c347608
Launching Web UI with arguments: --medvram --no-half --no-half-vae
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
Loading weights [4199bcdd14] from C:\Program Files\stable-diffusion-webui-directml\models\Stable-diffusion\revAnimated_v122.safetensors
Creating model from config: C:\Program Files\stable-diffusion-webui-directml\configs\v1-inference.yaml
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 9.0s (prepare environment: 2.1s, import torch: 2.5s, import gradio: 0.6s, setup paths: 0.6s, initialize shared: 1.0s, other imports: 0.3s, load scripts: 1.0s, create ui: 0.8s, gradio launch: 0.2s).
Loading VAE weights specified in settings: C:\Program Files\stable-diffusion-webui-directml\models\VAE\kl-f8-anime2.safetensors
Applying attention optimization: Doggettx... done.
Model loaded in 3.3s (load weights from disk: 0.5s, create model: 0.6s, apply weights to model: 1.5s, apply float(): 0.3s, load VAE: 0.1s, calculate empty prompt: 0.2s).
Canva size: 1280x960
Image size: 1280x960
Scale factor: 1
Tile size: 512x512
Tiles amount: 6
Grid: 2x3
Redraw enabled: True
Seams fix mode: NONE
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:18<00:00,  1.06it/s]
Total progress:  17%|██████████▊                                                      | 20/120 [00:16<01:15,  1.32it/s]
```

**Comment by rruki** [comment]
Can confirm that I also have the same issue. No error logs or anything.
Always stuck after the first iteration. Second iteration won't continue. webui terminal gets frozen. Have to force close.
Doesn't matter what sampling method is chosen, it will always hang at that same point.

**Comment by lshqqytiger** [comment]
May be fixed in 92849df26f73b416d396b95c3fb8c64070fe3ad8.

**Comment by Convexity-ai** [comment]
Commit 92849df resolved the issue for me. Thanks for the help.
